content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Enabling Multicluster on IBM Cloud Private
+description: Example multicluster IBM Cloud Private install of Istio.
+weight: 70
+keywords: [kubernetes,multicluster]
+---
+
+This example demonstrates how to use Istio' multicluster feature to join two
+[IBM Cloud Private](https://www.ibm.com/cloud/private) clusters together,
+using the [kubernetes multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/).
+
+## Create the IBM Cloud Private Clusters
+
+1.  You can follow steps [here](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_2.1.0.3/installing/installing.html)
+    to install two IBM Cloud Private clusters.
+    __NOTE__: Make sure individual cluster Pod CIDR ranges and service CIDR ranges must be unique across the multicluster
```

Comment:
must be unique->are unique and do not overlap

Reply:
Done